Framework/Algorithms/inc/MantidAlgorithms/ReflectometryWorkflowBase.h

```diff
@@ -52,18 +52,18 @@ class DLLExport ReflectometryWorkflowBase : public API::DataProcessorAlgorithm {
                    Mantid::API::MatrixWorkspace_sptr>;
   using OptionalInteger = boost::optional<int>;
 
-  /// Convert the input workspace to wavelength, splitting according to the
-  /// properties provided.
+  /// Splits the input workspace into detector and monitor workpsaces according
+  /// to the properties provided.
   DetectorMonitorWorkspacePair
   toLam(Mantid::API::MatrixWorkspace_sptr toConvert,
         const std::string &processingCommands,
         const OptionalInteger monitorIndex, const MinMax &wavelengthMinMax,
         const OptionalMinMax &backgroundMinMax, const double &wavelengthStep);
```

Comment:
And perhaps indicate in the descriptions that workspaces are expected to be in wavelength.